etc/metricshipper_supervisor.conf

```diff
@@ -1,5 +1,6 @@
 [program:metricshipper]
-command=metricshipper -c etc/metricshipper/metricshipper.yaml
+user=zenoss
+command=/opt/zenoss/bin/metricshipper -c /opt/zenoss/etc/metricshipper/metricshipper.yaml
 autorestart=true
 autostart=true
 startsecs=5
```

Comment:
is the default stopwaitsecs of 10 sufficient for metricshipper to complete any writes and shutdown cleanly?

Reply:
I don't see any issues in my dev environment with the default 10 sec value:
```
2019/09/10 08:44:29 200 4.881544ms POST /api/metrics/store
I0910 08:44:44.524028 00001 controller.go:704] Notifying subprocess of signal terminated for service b2m8nbvhgrkxs94fby7nmu0nb
I0910 08:44:44.524128 00001 controller.go:691] Closing healthExit on signal terminated for "b2m8nbvhgrkxs94fby7nmu0nb"
I0910 08:44:44.524152 00001 controller.go:693] Closed healthExit on signal terminated for "b2m8nbvhgrkxs94fby7nmu0nb"
I0910 08:44:44.524168 00001 controller.go:706] Notification complete for signal terminated for service b2m8nbvhgrkxs94fby7nmu0nb
2019-09-10 08:44:44,524 WARN received SIGTERM indicating exit request
2019-09-10 08:44:44,526 INFO waiting for metricshipper to die
2019-09-10 08:44:44,526 INFO stopped: metricshipper (terminated by SIGTERM)
2019/09/10 08:44:44 200 7.003626ms POST /api/metrics/store
2019/09/10 08:44:59 200 3.708743ms POST /api/metrics/store
```
But I'll increase it to 20 sec.